roles/config-rh-sso/tasks/setup-rh-sso.yml

```diff
@@ -3,11 +3,47 @@
   import_role:
     name: ../roles/config-packages
 
-- name: 'Update SSO bind addresses'
-  replace:
-    path: /etc/opt/rh/rh-sso7/keycloak/standalone/standalone.xml
-    regexp: '127.0.0.1'
-    replace: '0.0.0.0'
+- name: 'Update Red Hat SSO management/public interface IP address'
+  shell: >
+    "/opt/rh/rh-sso7/root/usr/share/keycloak/bin/jboss-cli.sh" -c
+    --command='/interface=management:write-attribute(name=inet-address,value="${jboss.bind.address:0.0.0.0}")'
+  shell: >
+    "/opt/rh/rh-sso7/root/usr/share/keycloak/bin/jboss-cli.sh" -c
+    --command='/interface=public:write-attribute(name=inet-address,value="${jboss.bind.address:0.0.0.0}")'
+  args:
+    executable: "/bin/bash"
+
+- name: 'Create a keystore to store SSL certificates'
+  java_keystore:
+    name: example
+    certificate: "{{lookup('file', 'path/to/the/file.cer') }}"
+    private_key: "{{lookup('file', 'path/to/the/file.key') }}"
+    password: "{{ sso_keystore_pass }}"
+    dest: /etc/security/keystore.jks
+
+- name: 'Add a new SSL based security-realm'
+  shell: >
+    "/opt/rh/rh-sso7/root/usr/share/keycloak/bin/jboss-cli.sh" -c
+    --command='/core-service=management/security-realm=UndertowRealm:add()'
+  args:
+    executable: "/bin/bash"
+  ignore_errors: yes
```

Comment:
Why `ignore_errors`? By default, we shouldn't be ignoring errors. 

Same goes for the ones below as well.

Reply:
I agree. This was meant to be temporary as I had to find a way to process some of the expected error codes, for instance, when I try to create the UndertowRealm and it's already defined exactly to the way I wanted, and still, Ansible was interrupting the play. This is nevertheless fixed.